.gitignore

```diff
@@ -17,3 +17,4 @@ __pycache__/
 /fuzz/Cargo.lock
 .mypy_cache/
 .pytest_cache/
+.idea/
```

Comment:
Is this something on your local machine? You can use the .git\info\exclude file to exclude local things without effecting the repo.